src/components/elements/typography.ts

```diff
@@ -46,6 +46,7 @@ type BaseElementProps = {
     max_height?: string
```

Comment:
Is this new Typography component or we will refactor this later?

Reply:
No this is the old one , we have created a new base element `TypographyBaseElement` which is extending this `BaseElement`. we can remove this once the code completely migrated to `Typography`

Reply:
cool, thank u @ashraf-deriv :+1: